api/handlers/v1/resource.go

```diff
@@ -3,82 +3,89 @@ package handlersv1
 import (
 	"context"
 	"errors"
-	"github.com/odpf/entropy/domain/model"
-	"github.com/odpf/entropy/domain/resource"
-	"github.com/odpf/entropy/service"
+	"github.com/odpf/entropy/container"
```

Comment:
@rohilsurana @whoAbhishekSah do we really need a container package?

Reply:
i think we can skip for now since we have only one repository and service. If needed we will add it later.